projects/public-search/src/app/document-detail/request/request.component.html

```diff
@@ -17,11 +17,11 @@
 <ng-container *ngIf="patron">
   <ng-container *ngIf="!requestDialog; else requestPickupDialog">
     <ng-container *ngIf="canRequest">
-      <div *ngIf="canRequest.can" id="item-request-{{ item.metadata.pid }}" class="row pl-4">
+      <div *ngIf="canRequest.can" id="record-request-{{ record.metadata.pid }}" [ngClass]="recordComponent == 'item' ? 'row pl-4' : ''" >
```

Comment:
same remark : maybe initiator component is badly structured (not critical)

Reply:
I thought is would be nicer for holding to have the button Request on the left rather than displaced on the right as it is done for the items.
<img width="1121" alt="Screenshot 2021-12-13 at 13 45 42" src="https://user-images.githubusercontent.com/42514476/145815349-25197d28-08fb-40ec-80f1-74586cb08b96.png">
Can I keep it like that ?

Reply:
@vgranata Is it possible to put the "Request" button at the end of the line and not below it, because it takes a lot of space?